src/FlaUI.WebDriver/Controllers/ElementController.cs

```diff
@@ -218,7 +218,8 @@ public async Task<ActionResult> ElementSendKeys([FromRoute] string sessionId, [F
 
             try
             {
-                await _actionsDispatcher.DispatchActionsForString(session, inputId, source, elementSendKeysRequest.Text);
+                //await _actionsDispatcher.DispatchActionsForStringUsingFlaUICore(session, inputId, source, elementSendKeysRequest.Text);
+                await _actionsDispatcher.DispatchSendKeysUsingFlaUICore(elementSendKeysRequest.Text);
```

Comment:
I don't really like having two branches of code for dispatching actions... Can't we replace the `DispatchActionsForString` implementation?